.evergreen/scripts/find-cmake-version.sh

```diff
@@ -30,6 +30,29 @@ make_tmpdir_in() {
   printf "%s" "${tmpdir}"
 }
 
+# Identify the cache directory to use.
```

Comment:
Diffs in this commit are mixed and mostly overwritten by the subsequent commit https://github.com/mongodb/mongo-c-driver/pull/1205/commits/fa0b1ad0c283f43540848b87d3c659297ff23470. Sorry for the confusion.